package/gluon-mesh-batman-adv/luasrc/lib/gluon/upgrade/310-gluon-mesh-batman-adv-mesh

```diff
@@ -23,6 +24,17 @@ uci:section('batman-adv', 'mesh', 'bat0', {
 })
 uci:save('batman-adv')
 
+uci:delete('batman-adv-legacy', 'bat0')
+uci:section('batman-adv-legacy', 'mesh', 'bat0', {
+	orig_interval = 5000,
+	gw_mode = gw_mode_legacy,
+	gw_sel_class = gw_sel_class,
+	hop_penalty = 15,
+	routing_algo = routing_algo,
+	multicast_mode = true,
+})
+uci:save('batman-adv-legacy')
+
```

Comment:
- These sections are applied using a procd hotplug trigger. bat0 should only be configured in batman-adv or batman-adv-legacy depending on the site config, never both
- multicast_mode doesn't make sense for -legacy